### PR TITLE
Make a few tweaks

### PR DESCRIPTION
**Pull request by nsg**
@petterarvidsson 

I have updated a lot of different things, see the commit messages for more info about a change. This is the changes that I like to do, but of course feel free to have opinions.

### CODE REVIEW

File: _includes/about.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2">
                 <h2>About Konstructs</h2>
-                <p>This is a Infiniminer/Minecraft inspired multiplayer
+                <p>This is a modded Minecraft/Minetest inspired multiplayer
```
Comment (petterarvidsson) [inline _includes/about.html:6]: @nsg What does "modded" mean?

**Review by petterarvidsson — APPROVED** [review]
:+1: Left a comment and made some changes.

File: _includes/about.html (hunk 1 above)
Comment (nsg) [reply _includes/about.html:6]: mod, modification and modified minecraft is usually called modded minecraft. A common lingo.

**Comment by nsg** [comment]
Thank you for the fixes, merge time!

**Pull request merged**